packages/utilities/src/scroll.ts

```diff
@@ -146,8 +146,9 @@ export function getScrollbarWidth(): number {
 }
 
 /**
- * Traverses up the DOM for the element with the data-is-scrollable=true attribute, or returns
- * document.body.
+ * Traverses up the DOM for the element with the data-is-scrollable=true attribute and parallelly
+ * checks if the overflow property of an element allows the element to scroll (while excluding
+ * the elements with absolute, static and fixed positioning) , or finally returns document.body.
```

Comment:
I'm primarily worried about regression of `findScrollableParent` without any test coverage since it seems like we're modifying behavior here. cc: @ThomasMichon for awareness.

It also looks like we're initializing a `RegExp` within a tight loop below which I thought incurred _some_ performance cost.